scapy/arch/consts.py

```diff
@@ -0,0 +1,26 @@
+## This file is part of Scapy
+## See http://www.secdev.org/projects/scapy for more informations
+## Copyright (C) Philippe Biondi <phil@secdev.org>
+## This program is published under a GPLv2 license
+
+import os
+from sys import platform
+
+LINUX = platform.startswith("linux")
+OPENBSD = platform.startswith("openbsd")
+FREEBSD = "freebsd" in platform
+NETBSD = platform.startswith("netbsd")
+DARWIN = platform.startswith("darwin")
+SOLARIS = platform.startswith("sunos")
+WINDOWS = platform.startswith("win32")
+BSD = DARWIN or FREEBSD or OPENBSD or NETBSD
+
+if WINDOWS:
+    X86_64 = False
```

Comment:
Actually it was previously set to `False` and is only used on Unix flavors (correction: only on Linux actually)... But that's a bit misleading, you're right.